### ISSUE TITLE
[BUG] Slackbot notifier false positive notification 

### ISSUE THREAD

**Issue by etagwerker** [fastruby/pecas#125]
## Expected Behavior

When I log a time entry like this one: 

> Worked on PROJECT-1235: Solved bug and submitted PR 

I should not get a notification from the Slackbot. 

## Actual Behavior

When I log a time entry like this one: 

> Worked on PROJECT-1235: Solved bug and submitted PR 

I get a notification from the Slackbot. 

## Possible Fix

Improve the regular expression. The `:` is throwing things off. 

## To Reproduce

1. Log an entry with `PROJECT-123:` in Noko 
2. Get a notification around 8pm 

## Additional Information

Here is more info: 

- https://github.com/fastruby/pecas/blob/b8891addcd2d9b5d53b936621ac531a216098fa3/app/domain/time_entry/description_rules.rb#L18-L24
- https://github.com/fastruby/pecas/blob/b8891addcd2d9b5d53b936621ac531a216098fa3/app/domain/time_entry/description_rules.rb#L2

**I will abide by the [code of conduct](https://github.com/fastruby/pecas/blob/main/code-of-conduct.md)**

**Comment by arielj** [comment]
Could be useful to also check this line doing a `gsub` before matching the Jira ID:
https://github.com/fastruby/pecas/blob/b8891addcd2d9b5d53b936621ac531a216098fa3/app/domain/time_entry/description_rules.rb#L42

Because will have a similar issue doing something like `Woking on something (PROJ-123)` since the `(` `)` will also make it fail.

Not sure what's better, if an update of the `gsub` regular expression removing chars or if an update on the JIRA regular expression to ignore unwanted chars.

**Comment by aisayo** [comment]
I agree that updating the `gsub` to removing chars will be more beneficial.
